test/fixtures/user/user.js

```diff
@@ -54,7 +54,6 @@ module.exports = () => {
       twitter_id: "whatifi",
       discordJoinedAt: "2023-04-06T01:47:34.488000+00:00",
       phone: "1234567891",
-      email: "abc1@gmail.com",
```

Comment:
Removed this to test one of the case, on adding new fixture one of the other test case was failing, so modified the existing one.